src/friendly-iframe-embed.js

```diff
@@ -180,17 +215,8 @@ export function installFriendlyIframeEmbed(
   iframe.setAttribute('marginheight', '0');
   iframe.setAttribute('marginwidth', '0');
 
-  // Compute extensions.
-  const extensionIds =
-    // TODO(#33020): Remove extensionIds format once it's supported everywhere.
-    spec.extensionIds
-      ? spec.extensionIds
-      : spec.extensions
-      ? spec.extensions.map(({extensionId}) => extensionId)
-      : [];
-
   // Pre-load extensions.
-  preloadFriendlyIframeEmbedExtensionIdsDeprecated(win, extensionIds);
+  preloadFriendlyIframeEmbedExtensions(win, spec.extensions || []);
```

Comment:
Please keep the old code in place for now. There are couple more changes I need to make to allow versioned installation. I will switch these out myself.

Reply:
You have written it so that `preloadFriendlyIframeEmbedExtensions` calls `preloadFriendlyIframeEmbedExtensionsDeprecated` but this pr makes it so that is the only caller. Is that ok?

Reply:
Yep. That's exactly what I want. That will allow me to "turn on" multi-versioned mode in one pull request everywhere. I think that means that you should be able to simply revert this file completely.

Reply:
Resolved offline, this is not yet using the extension versions in calls to extension service.